stable/spinnaker/templates/statefulsets/halyard.yaml

```diff
@@ -66,7 +70,11 @@ spec:
       {{- if and .Values.s3.enabled .Values.s3.accessKey .Values.s3.secretKey }}
```

Comment:
Here also you'd have to include .Values.s3.secretName

Reply:
@vyasswaroop forgot to put it [here](https://github.com/helm/charts/blob/b3c167d9f92bad05118ecfce865d0f0a238b7914/stable/spinnaker/templates/configmap/halyard-config.yaml#L53) too, thanks.